app/services/get-breadcrumbs.js

```diff
@@ -27,37 +28,53 @@ const getBreadcrumbLevels = function () {
 
 const getOffenderManagerBreadcrumb = function (id, workloadType) {
   return getWorkloadOwnerOffenderManagerByWorkloadOwnerId(id).then(function (offenderManager) {
-    const breadcrumbs = [new Link(`${offenderManager[0].offender_manager_forename} ${offenderManager[0].offender_manager_surname}`, `/${workloadType}/offender-manager/${id}`)]
-    return getTeamBreadcrumb(offenderManager[0].team_id, workloadType).then(function (teamBreadcrumbs) {
-      return breadcrumbs.concat(teamBreadcrumbs)
-    })
+    if (offenderManager.length) {
```

Comment:
bit of a strange place to put all of this, this is the first time any level is retrieved, the alternative solution is to swap getBreadcrumbs everywhere in the routes to the last call in the chain and have a not found check there